packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -112,10 +110,12 @@ const chatMessageStyles: ComponentSlotStylesInput<
   content: ({ props: p, variables: v }): ICSSInJSStyle => ({
     color: v.contentColor,
     display: 'block',
-    '& a:focus': {
+    '& a': {
```

Comment:
@codepretty can this be defined as a general rule? I am pretty sure all links in the app should be consistent...

Reply:
i think so too. as long as this doesn't affect any links in dropdowns inside a chatmessage?

Reply:
closing now, we can revisit in dedicated PR when it comes to styling links